examples/src/main/java/io/harness/ff/examples/ExcessiveLogs.java

```diff
@@ -0,0 +1,69 @@
+package io.harness.ff.examples;
+
+import com.google.gson.JsonObject;
+import io.harness.cf.client.api.*;
+import io.harness.cf.client.connector.HarnessConfig;
+import io.harness.cf.client.connector.HarnessConnector;
+import io.harness.cf.client.dto.Target;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.MDC;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
+public class ExcessiveLogs {
+//    private static final String SDK_KEY = "97432269-f803-4efa-bca9-36c22d77ad5d";
+    private static final String SDK_KEY = "2e182b14-9944-4bd4-9c9f-3e859e2a2954";
```

Comment:
Is this a real key?  If so it might be worth replacing.

Reply:
removed from git, it is just for easier testing